docker/.dockerignore

```diff
@@ -1,6 +1,8 @@
-Dockerfile
-.dockerignore
```

Comment:
just checking, do we still want to ignore the Dockerfile and .dockerignore, or is below all we need?

Reply:
I recently learned that ignoring these can result in odd/undefined behavior. So I dropped those.

Docker also expects some weirdness when the context is set the way I have it (one dir above next-build so that vets-website can be part of it), so that's also why I'm doing the cp shenanigans in the workflow. All sorts of oddness I wasn't previously aware of!

Reply:
interesting, TIL. thanks!